runtime/frost/src/lib.rs

```diff
@@ -417,13 +417,11 @@ parameter_types! {
 }
 
 impl pallet_transaction_payment::Config for Runtime {
-	type OnChargeTransaction = CurrencyAdapter<Balances, DealWithFees>;
```

Comment:
This is undoing the required change for fee handling. Please line from previous commit

Reply:
Ok sure